plugin/adsk/scripts/mayaUsd_help.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Autodesk
```

Comment:
Same comment as the helpTable, this function should be in mayaUsd instead of the plugin. I would have just added your function to the existing "lib/mayaUsd/resources/scripts/mayaUsdUtils.py" file. But if you want to have it in a separate file in that folder that is also okay.